library does not build with jdk 9

...
[ERROR] Failed to execute goal com.buschmais.jqassistant.scm:jqassistant-maven-p
lugin:1.1.4:scan (default) on project security-logging: Execution default of goa
l com.buschmais.jqassistant.scm:jqassistant-maven-plugin:1.1.4:scan failed: Unab
le to load the mojo 'scan' in the plugin 'com.buschmais.jqassistant.scm:jqassist
ant-maven-plugin:1.1.4'. A required class is missing: javax/activation/DataSource

--- a/pom.xml
+++ b/pom.xml
@@ -104,9 +104,8 @@
         </executions>
       </plugin>
 	<plugin>
-            <groupId>com.buschmais.jqassistant.scm</groupId>
+            <groupId>com.buschmais.jqassistant</groupId>
             <artifactId>jqassistant-maven-plugin</artifactId>
-            <version>1.1.4</version>
             <executions>
                 <execution>
                     <goals>

